pkg/gpu/nvidia/beta_plugin.go

```diff
@@ -82,13 +82,11 @@ func (s *pluginServiceV1Beta1) Allocate(ctx context.Context, requests *pluginapi
 			})
 		}
 
-		for _, mountPath := range s.ngm.mountPaths {
-			resp.Mounts = append(resp.Mounts, &pluginapi.Mount{
-				HostPath:      mountPath.HostPath,
-				ContainerPath: mountPath.ContainerPath,
-				ReadOnly:      true,
-			})
+		for i := range s.ngm.mountPaths {
```

Comment:
line 60 in this file, the package name is `timesharing.ValidateRequest`, but it applies to both MPS and timesharing (if the virtual ID naming rule is the same), we'll need to update the package name as a follow-up

Reply:
Forgot about it, thanks for pointing it :).